README.md

```diff
@@ -396,14 +396,71 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example) and clears all tokens stored in the `TokenManager`.
+
+By default:
+
+  - the user will be signed out using an API method 
+  - the page location will not be changed. 
+  
+When calling `signOut` with the default options, it is recommended to reload the page after the promise resolves to ensure all user data is cleared from memory and the app is properly loaded in an unauthenticated state.
```

Comment:
Are we recommending this because of our potential stale state, or are we recommending this because their app may have its own stale state?  

Will a page reload suffice if any stale state is in localStorage/cookies?

Reply:
Recommending it for their stale state and UI. If we have left tokens in localStorage or cookies, those will survive the reload (but we will not do this). 

`signOut` should be the last thing the app instance does using the user context. Our redirect scenario will do this naturally for them. but if you choose not to redirect, doing a reload() after signOut is "generally" good practice.

Advanced users can decide what is best for their app, of course. If authentication is a small piece of the app, not central to the experience, maybe MAYBE it does not need to reload. Maybe they are doing a user-switcher and they want to logout and log back in without a page reload? I think it's very advanced/edgy. 

Typically the user would expect to see the unauthenticated state after logging out.